cmd/titus-metadata-service/main.go

```diff
@@ -118,6 +118,7 @@ func main() {
 		backingMetadataServer      string
 		metatronEnabled            bool
 		region                     string
+		accountID                  string
```

Comment:
Where does `accountID` get passed into `types.MetadataServerConfiguration` as `NetflixAccountID`?  Should it be around L246 below?